book/src/drivers.md

```diff
@@ -40,6 +40,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Nutanix](https://portal.nutanix.com/#/page/docs/details?targetId=CSI-Volume-Driver:CSI-Volume-Driver) | `"com.nutanix.csi"` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Nutanix | Persistent | "Read/Write Single Pod" with Nutanix Volumes and "Read/Write Multiple Pods" with Nutanix Files | Yes | No | No | ? | ?
 [OpenSDS](https://github.com/opensds/nbp/tree/master/csi) | `csi-opensdsplugin` | v1.0 | A Container Storage Interface (CSI) Driver for  [OpenSDS]((https://www.opensds.io/)) | Persistent | Read/Write Single Pod | Yes | Yes | Yes | ? | ?
 [Portworx](https://github.com/libopenstorage/openstorage/tree/master/csi) | `pxd.openstorage.org` | v0.3, v1.1 | A Container Storage Interface (CSI) Driver for [Portworx](https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/csi/) | Persistent | Read/Write Multiple Pods | Yes | No | Yes | Yes | ?
+[Pure Storage CSI](https://github.com/purestorage/helm-charts)| `pure-csi-plugin` | v1.0 | A Container Storage Interface (CSI) Driver for Pure Storage's [Pure Service Orchestrator](https://purestorage.com/containers) | Persistent | "Read/Write Single Pod" with FlashArray and "Read/Write Multiple Pods" with FlashBlade| Yes | No | No | ? | ?
```

Comment:
Can you fill in the last 2 question marks

Reply:
Done